### PR TITLE
added bulk event update in the schedule view

### PR DESCRIPTION
**Pull request by WilielFlorenciani**

### CODE REVIEW

File: src/Schedule/UpdateBatchEvent.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+            let StartTimeInMilliSeconds = +new Date(defaultStartTime);
+        await axios.put(`api/showroom/schedule/events/bulk/${StartTimeInMilliSeconds}`, messageData, {
+            headers: {"Content-Type": "application/json"},
+            withCredentials: true
+            }).then((res) => {
+                console.log(res.data)
+            }).catch((error)=>{
+                console.log(error)
+        });
+        history.push('/cal');
```
Comment (DanielTorres10) [inline src/Schedule/UpdateBatchEvent.js]: history.push(`/schedule/${cid}/eventsScheduled`);

File: src/Schedule/UpdateBatchEvent.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    
+        await axios.delete(`api/showroom/schedule/events/${eid}`,  {
+            headers: {"Content-Type": "application/json"},
+            withCredentials: true
+            }).then((res) => {
+                console.log(res.data)
+            }).catch((error)=>{
+                console.log(error)
+        });
+        history.push('/cal');
```
Comment (DanielTorres10) [inline src/Schedule/UpdateBatchEvent.js]: history.push(`/schedule/${cid}/eventsScheduled`);

**Review by DanielTorres10 — COMMENTED** [review]
update /cal path

**Review by WilielFlorenciani — COMMENTED** [review]
checked files

**Pull request merged**